app/src/main/java/com/ciandt/dragonfly/example/data/local/entities/ProjectEntity.kt

```diff
@@ -0,0 +1,15 @@
+package com.ciandt.dragonfly.example.data.local.entities
+
+import android.arch.persistence.room.Entity
+import android.arch.persistence.room.Ignore
+import android.arch.persistence.room.PrimaryKey
+
+@Entity(tableName = "projects")
+data class ProjectEntity(
```

Comment:
No 😞 , because we need set/get to java access.
Look to the generated code:

```java
_tmpProject = new ProjectEntity();
final String _tmpId;
_tmpId = _cursor.getString(_cursorIndexOfId);
_tmpProject.setId(_tmpId);
final String _tmpName;
_tmpName = _cursor.getString(_cursorIndexOfName);
_tmpProject.setName(_tmpName);
final String _tmpDescription;
_tmpDescription = _cursor.getString(_cursorIndexOfDescription);
_tmpProject.setDescription(_tmpDescription);
final String _tmpColors;
_tmpColors = _cursor.getString(_cursorIndexOfColors);
_tmpProject.setColors(_tmpColors);
final long _tmpCreatedAt;
_tmpCreatedAt = _cursor.getLong(_cursorIndexOfCreatedAt);
_tmpProject.setCreatedAt(_tmpCreatedAt);
```